src/threema/container.ts

```diff
@@ -711,24 +711,25 @@ class Messages implements threema.Container.Messages {
         return false;
     }
 
-    public bindTemporaryToMessageId(receiver: threema.BaseReceiver, temporaryId: string, messageId: string): boolean {
+    public bindTemporaryToMessageId(
```

Comment:
Can you add a docstring? I had to read through the function body to remember what the function does.